certified-connectors/Zoho ZeptoMail/readme.md

```diff
@@ -0,0 +1,35 @@
+# Zoho ZeptoMail
```

Comment:
@zohomail can you please include all the required section from our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for code raw view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).